src/cargo/core/shell.rs

```diff
@@ -25,6 +25,16 @@ pub enum ColorConfig {
     Never
 }
 
+impl ToString for ColorConfig {
+    fn to_string(&self) -> String {
+        match *self {
+            ColorConfig::Auto => "auto",
+            ColorConfig::Always => "always",
+            ColorConfig::Never => "never",
+        }.to_string()
+    }
+}
```

Comment:
Oh actually, this should implement `fmt::Display` conventionally instead of `ToString` directly.